src/workflow/workflow.js

```diff
@@ -267,6 +290,8 @@ const pinAndSetImageCid = async (wfConfig) => {
   if (isUpdated) {
     context.data.setColumns([imageCidColumn, metaCidColumn]);
     if (!dryRun) context.data.checkpoint();
+  } else {
+    console.log('Re-using data from the checkpoint');
```

Comment:
It's not only the checkpoint, e.x. the offset/count can return zero rows. Maybe say "No metadat was updated." or we can change the isUpdated to be a counter and then print how many metadatas was set?!